doc/design.md

```diff
@@ -94,31 +95,41 @@ Covers:
 
 Needs: impl, utest, itest
 
-## `UserInformation` Caches User Roles
-`dsn~user-information-caches-user-roles`
-
-The `UserInformation` caches the roles a user has.
+## Query Rewriter Determines User Roles
```

Comment:
```suggestion
## `QueryRewriter` Determines User Roles
```

Please repeat for all other occurrences.

Reply:
fixed